component/authenticator/src/main/java/org/wso2/carbon/identity/authenticator/x509Certificate/X509CertificateConstants.java

```diff
@@ -41,13 +41,16 @@ private X509CertificateConstants() {
     public static final String AUTHENTICATION_ENDPOINT = "https://localhost:8443/x509-certificate-servlet";
     public static final String AUTHENTICATION_ENDPOINT_PARAMETER = "AuthenticationEndpoint";
     public static final String USERNAME = "username";
+    public static final String ENFORCE_SELF_REGISTRATION = "EnforceSelfRegistration";
     public static final String SUCCESS = "success";
     public static final String RETRY_PARAM_FOR_CHECKING_CERTIFICATE =
             "&authFailure=true&errorCode=";
     public static final String ERROR_PAGE = "x509certificateauthenticationendpoint/x509CertificateError.jsp";
     public static final String CLAIM_URI = "setClaimURI";
     public static final String AUTHENTICATORS = "authenticators";
     public static final String X509_CERTIFICATE_ERROR_CODE = "X509CertificateErrorCode";
-    public static final String X509_CERTIFICATE_NOT_FOUND_ERROR_CODE = "404";
+    public static final String X509_CERTIFICATE_NOT_FOUND_ERROR_CODE = "500";
```

Comment:
These are very specific HTTP error codes. Let's try to use the error codes from our Error code schema.

Reply:
Fixed in 548fb381552eab88ce59ebe0e845891ba73c1fa1